src/Ushahidi/Modules/V3/ServiceProvider.php

```diff
@@ -2,27 +2,27 @@
 
 namespace Ushahidi\Modules\V3;
 
-use Ushahidi\Core\Tool\Verifier;
-use Ushahidi\Modules\V3\Console;
 use Illuminate\Support\Facades\Route;
+use Illuminate\Support\ServiceProvider as BaseServiceProvider;
+use Ushahidi\Modules\V3\Console;
+use Ushahidi\Modules\V3\Factory\UsecaseFactory;
+use Ushahidi\Modules\V3\Http\Middleware\RepositoryBinder;
+use Ushahidi\Core\Tool\Verifier;
 use Ushahidi\Core\Usecase\User\LoginUser;
 use Ushahidi\Core\Usecase\Post\ExportPost;
-use Ushahidi\Core\Usecase\Export\Job\PostCount;
-use Ushahidi\Modules\V3\Factory\UsecaseFactory;
 use Ushahidi\Core\Usecase\Message\ReceiveMessage;
-use Ushahidi\Modules\V3\Repository\TosRepository;
+use Ushahidi\Core\Usecase\Export\Job\PostCount;
+use Ushahidi\Contracts\Repository\Entity\TagRepository;
```

Comment:
@webong Isn't this the same typo you fixed here? https://github.com/ushahidi/platform/pull/4666/files

Reply:
Absolutely is! 🤦🏼‍♂️